tests/phpunit/CRM/Core/BAO/MessageTemplateTest.php

```diff
@@ -382,7 +384,7 @@ public function testContactTokens(): void {
    */
   public function hookTokenValues(array &$details): void {
     foreach ($details as $index => $detail) {
-      $details[$index]['favourite_emoticon'] = 'emo';
+      $details[$index]['important_stuff.favourite_emoticon'] = 'emo';
```

Comment:
@eileenmcnaughton  Is this a contract change? On one hand, it smells sorta like it on the surface. OTOH, this part of the unit-test seems to be fairly recent, and there should be other places with more coversage. Is this a place where maybe the test was incomplete/passing-for-the-wrong-reason before?

Reply:
I think the test was wrong - I only just added it in the context of making this change but then it passed & all the other token tests failed..... so I figured I had gotten it wrong

Reply:
From docs


![image](https://user-images.githubusercontent.com/336308/133202099-1f17b0f6-9e60-40c1-b0e4-8f1187a42a3d.png)

Reply:
Yeah & also [here](https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_tokenValues/)

Reply:
Cool. Yeah, I think that's right (the revision seems to match-up more with the [devdocs](https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_tokenValues/#example) and other tests).